src/Tasks/Common/Resources/Strings.resx

```diff
@@ -118,202 +118,267 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="AtLeastOneTargetFrameworkMustBeSpecified" xml:space="preserve">
-    <value>At least one possible target framework must be specified.</value>
+    <value>NETSDK1001: At least one possible target framework must be specified.</value>
```

Comment:
Let's add tests that validate that we don't duplicate error codes over time.

To ensure that this still holds even for deleted messages, we discussed having a list of deleted message codes, and ensuring that there are no duplicates between the current and deleted message codes, and that the current and deleted message codes are contiguous.

Reply:
Done.